### PR TITLE
Release 0.3

### PR DESCRIPTION
**Pull request by bilgili**

### CODE REVIEW

File: doc/Changelog.md [hunk 1]
```diff
@@ -3,6 +3,8 @@ Changelog {#Changelog}
 
 # git master {master)
 
+# Release 0.3.0 (09-11-2015) {#release_003000}
+
 * [6](https://github.com/BlueBrain/Monsteer/pull/6):
```
Comment (rdumusc) [inline doc/Changelog.md]: missing a # here for consistency...

**Comment by rdumusc** [comment]
+1 otherwise

File: .gitsubprojects [hunk 2]
```diff
@@ -2,4 +2,4 @@
 git_subproject(Brion https://github.com/BlueBrain/Brion.git 015515f)
 git_subproject(vmmlib https://github.com/Eyescale/vmmlib.git 2bec113)
 git_subproject(Lunchbox https://github.com/Eyescale/Lunchbox.git f48795e)
```
Comment (hernando) [inline .gitsubprojects]: Update this one

Comment (bilgili) [inline .gitsubprojects]: I have managed to compile with the mimimum change, thats why I kept.

File: doc/Changelog.md (hunk 1 above)
Comment (eile) [inline doc/Changelog.md]: Remove the anchors on the release headings (see other projects) - They look ugly on github and don't add much for doxygen.

File: .gitsubprojects (hunk 2 above)
Comment (eile) [inline .gitsubprojects]: We updated them in all other projects to the released versions - in any case you'll get new ones.

File: CMake/CPackConfig.cmake [hunk 3]
```diff
@@ -9,5 +9,5 @@
 
 set(CPACK_PACKAGE_DESCRIPTION_FILE "${PROJECT_SOURCE_DIR}/README.md")
 set(CPACK_DEBIAN_PACKAGE_DEPENDS "libboost-test-dev")
-
+SET(CPACK_PACKAGE_LICENSE "LGPL")
```
Comment (eile) [inline CMake/CPackConfig.cmake]: No - updated GitExternals will use MONSTEER_LICENSE from top-level CMakeLists.

**Comment by bilgili** [comment]
ping

**Comment by rdumusc** [comment]
+1 again :-)

**Pull request merged**